backend/src/services/integration-auth/integration-app-list.ts

```diff
@@ -455,6 +455,28 @@ const getAppsCircleCI = async ({ accessToken }: { accessToken: string }) => {
   return apps;
 };
 
+/**
+ * Return list of projects for Databricks integration
+ */
+const getAppsDatabricks = async ({ accessToken }: { accessToken: string }) => {
+  const res = await request.get<{ scopes: { name: string; backend_type: string }[] }>(
+    `${IntegrationUrls.DATABRICKS_API_URL}/2.0/secrets/scopes/list`,
```

Comment:
I think we should be passing in the databricks workspace url from the integration as it's unique to each instance